orchestration/dagster_orchestration/hca_orchestration/tests/test_solids.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+from unittest.mock import Mock, patch
+
+from dagster import execute_solid
+
+from hca_orchestration.solids import post_import_validate
+
+
+class SolidsTestCase(unittest.TestCase):
+    @patch("hca_utils.utils.HcaUtils.get_null_filerefs")
+    @patch("hca_utils.utils.HcaUtils.get_file_table_names")
+    @patch("hca_utils.utils.HcaUtils.get_duplicates")
+    @patch("hca_utils.utils.HcaUtils.get_all_table_names")
```

Comment:
the fact that we need to patch this many things to mock out our interactions with jade is a code smell to me, but this probably isn't the pr to address that. do we have a tech debt ticket to make mocking `HcaUtils` more turnkey?

Reply:
I can add one; I am v new to the whole mock library so this is helpful! Would the alternative be to like make a `MockHcaUtils` class or something and just reuse that?

Reply:
I added a ticket to write tests for hca utils https://broadinstitute.atlassian.net/browse/DSPDC-1570